### PR TITLE
adding task to update replicas

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/12705

### CODE REVIEW

File: observability/customize_observability.adoc [hunk 1]
```diff
@@ -221,6 +222,30 @@ spec:
...
 For descriptions of all the parameters that can added into the `advanced` configuration, see link:../apis/observability.json.adoc[Observability API].
 
+[#updating-replicas]
+== Updating observability replicas
+
+If your workload increases, increase the number of replicas of your observability pods. Complete the following steps to update your replicas:
+
+. Log in to your {product-title-short} cluster.
+. From the console header, click the _Applications_ button > *OpenShift Container Platform*.
+. From the {ocp-short} navigation menu, select *StatefulSets*.
```
Comment (clyang82) [inline observability/customize_observability.adoc]: Do you mean update the mco cr from UI?

Comment (dockerymick) [reply observability/customize_observability.adoc]: Yea, i mentioned the mco cr in the next step. Should I change the section title from Updating observability replicas to **Updating the mco CR from the console**?

Comment (clyang82) [reply observability/customize_observability.adoc]: Yes. I think so. otherwise the customer may not have console context

Comment (clyang82) [reply observability/customize_observability.adoc]: If you want to user updating the mco CR from the console, then there should not run `oc edit mco observability -o yaml`.

Here is my suggested steps: 

. Log in to your {product-title-short} cluster.
. From the console header, click the _Applications_ button > *OpenShift Container Platform*.
. From the {ocp-short} navigation menu, select *Administration*->*CustomResourceDefinitions*
. Search for `multiclusterobservability`. from *Instances* tab, select `observability` instance
. You can edit the yaml file in *YAML* tab. Your updated YAML might resemble the following content:
```
spec:
   advanced:
      receive:
         replicas: 6
```

**Comment by dockerymick** [comment]
@clyang82 thanks for your feedback! When you have an opportunity may you review the PR? I updated the file based on your suggestions.

**Comment by clyang82** [comment]
/lgtm
Thanks @dockerymick

**Pull request merged**